server/myDB/user.go

```diff
@@ -0,0 +1,47 @@
+package myDB
+
+import (
+	"github.com/jinzhu/gorm"
+	"golang.org/x/crypto/bcrypt"
+)
+
+type User struct {
+	gorm.Model
+	FirstName string `json:"firstName"`
+	LastName  string `json:"lastName"`
+	Username  string `json:"username"`
+	Password  string `json:"password"`
+	Email     string `json:"email"`
+	PhoneNum  string `json:phoneNum`
+}
+
+func CreateUser(user *User, db gorm.DB) {
+	db.Debug().Create(user)
+}
+func CheckCredentials(username string, password string, db gorm.DB) bool {
+	user := User{}
+	db.Debug().Where("username = ? ", username).Find(&user)
+
+	if "" != user.Username {
```

Comment:
Is there a better way to do the comparison. How about we change it up to this?
```suggestion
	if user.Username.empty {
```